config/helm/appmesh-controller/templates/deployment.yaml

```diff
@@ -113,6 +113,7 @@ spec:
         - --sidecar-log-level={{ .Values.sidecar.logLevel }}
         # this must be same as livenessProbe port which can be configured 
         - --health-probe-port={{ .Values.livenessProbe.httpGet.port }}
+        - --wait-until-proxy-starts={{ .Values.sidecar.waitUntilProxyStarts }}
```

Comment:
is it until-start or until-ready? Do we wait for Envoy to be bootstrapped with config from App Mesh XDS?

Reply:
yeah i can see until-ready to be a more apt name. yes it is supposed to wait until the sidecars have the mesh configs

Reply:
I second that. we should probably change the parameter name to be more consistent with the condition we are looking for.